SamplifyAPIClientTest/Helper.cs

```diff
@@ -5,54 +5,58 @@ namespace SamplifyAPIClientTest
 {
     public static class Helper
     {
+
         public static ProjectCriteria GetTestProject()
         {
+            Int32 timestamp = (Int32)(DateTime.UtcNow.Subtract(new DateTime(1970, 1, 1))).TotalSeconds;
             return new ProjectCriteria
             {
-                ExtProjectID = "project001",
+                ExtProjectID = "project" + timestamp,
```

Comment:
use StringBuilder to append string - more efficient

Reply:
Fixed